test/targets.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { SlippiGame, TargetTestResultType } from "../src";
+
+describe("when processing break the target replays", () => {
+  it("should correctly count the target breaks", () => {
```

Comment:
Can we also include a test that the target breaks array is empty if it's not a target test game replay?